packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -200,10 +200,11 @@ export class Combobox
    * Specifies the selection mode:
    * - `multiple` allows any number of selected items (default),
    * - `single` allows only one selection,
+   * - `single-persist` is like single, but does not allow deselecting,
```

Comment:
nitpick:

For consistency, can we add double quotes to each of the values? For instance:

```
- `"multiple"`....
- `"single"`....
- `"single-persist"`...
- etc.
```